webapp/src/RepoGit.ts

```diff
@@ -158,3 +176,9 @@ export class RepoGit {
     return paths;
   }
 }
+
+export async function getRepoGit(
```

Comment:
Why we need to export this function instead of use static function getRepoGit of the class RepoGit?

Reply:
I added the exported function for terser call sites. For example, `Cache.ts` can await `getRepoGit` instead of awaiting `RepoGit.getRepoGit`.